app/javascript/controllers/analytics_controller.js

```diff
@@ -0,0 +1,11 @@
+import { Controller } from "@hotwired/stimulus"
+
+export default class extends Controller {
+  connect() {
+    const debug = Boolean(document.head.querySelector("meta[name=analytics_debug]").getAttribute("value"))
+    window.dataLayer = window.dataLayer || [];
+    function gtag(){dataLayer.push(arguments);}
+    gtag('js', new Date());
+    gtag('config', 'G-48Y3Q1L4K0', {'debug_mode': debug});
```

Comment:
i think @jcoyne mentioned elsewhere that we should turn on [IP address anonymization for analytics](https://support.google.com/analytics/answer/2763052?hl=en). looks like that's as simple as:
```suggestion
    gtag('config', 'G-48Y3Q1L4K0', {'debug_mode': debug, 'anonymize_ip': true});
```

Reply:
nice!